lms/djangoapps/courseware/views/views.py

```diff
@@ -736,8 +751,18 @@ def course_about(request, course_id):
         if configuration_helpers.get_value('ENABLE_MKTG_SITE', settings.FEATURES.get('ENABLE_MKTG_SITE', False)):
             return redirect(reverse(course_home_url_name(course.id), args=[unicode(course.id)]))
 
+        from ast import literal_eval
```

Comment:
please move this import to the top level

Reply:
@dyudyunov check this ast helpers) it's good to know them) https://docs.python.org/3/library/ast.html#module-ast